.github/workflows/python-package.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
```

Comment:
This could probably do with a more informative name, maybe something like "Build and Test" or "Run Tests"?